plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -27,11 +27,15 @@ open class HtmlRenderer(
         node: ContentGroup,
         pageContext: ContentPage,
         childrenCallback: FlowContent.() -> Unit
-    ) = when {
-        node.dci.kind == ContentKind.BriefComment -> div("brief") { childrenCallback() }
-        node.style.contains(TextStyle.Paragraph) -> p { childrenCallback() }
-        node.style.contains(TextStyle.Block) -> div { childrenCallback() }
-        else -> childrenCallback()
+    ) {
+        val additionalClasses = if (node.style.contains(TextStyle.Monospace)) "monospace" else ""
```

Comment:
Instead of this, I'd suggest doing this:
```suggestion
        val additionalClasses =  node.style.joinToString(" ") { it.toString().toLowerCase() }
```
This way we'll have all the classes and we can always not use them in the css

Reply:
ok